Electronics/PCB/AstroPlant_ExtensionBoard/astroplant-extension-board-mk0-6-1/README.md

```diff
@@ -0,0 +1,42 @@
+
+#If you want to generate output files, follow these steps: 
```

Comment:
Some notes after I tested it on my laptop:


gerber files are almost identical, the differences are many, but usually just one digit per line, like this:
```
 X192991090Y-84128502D01*
 X193012064Y-84145405D01*
-X193154594Y-84287935D01*
-X193188620Y-84350247D01*
-X193191499Y-84377030D01*
+X193154595Y-84287936D01*
+X193188621Y-84350248D01*
+X193191500Y-84377031D01*
 X193191500Y-91397968D01*
 X193171498Y-91466089D01*
```
 can you double check this is correct?

Drill files are identical except dates

Reply:
These are placement coordinates, so I can't say whether the new coordinates generated are good or bad - they need to be tested. That's why it's risky to have anyone generate files.